### ISSUE TITLE
How to update atuin ?

### ISSUE THREAD

**Issue by baptistebisson** [atuinsh/atuin#1169]
Hello.
I might have miss something but didn't found it in the doc: how to update atuin ?

I've been doing the bash/curl command but is there an update command available ? With the curl, every time I need to remove `source ~/.atuininit.sh` from my .zshrc file.

Thanks.

**Comment by cameronj86** [comment]
Also googling to find the answer to this as well. Getting the prompt for 16 but still waiting on a workaround. 

Installed 16 on my laptop and it appears to honor my disabling of the Up_Key better than 15 did already, so looking forward to having it on my desktop

**Comment by ellie** [comment]
There's no update command as most users will have it installed via the system package manager - the install script just uses that

If you can't update with your package manager, running the install script again + cleaning up your config is the best way at present

**Comment by baptistebisson** [comment]
> There's no update command as most users will have it installed via the system package manager - the install script just uses that
> 
> If you can't update with your package manager, running the install script again + cleaning up your config is the best way at present

Script is the recommended way in the documentation and there is no debian/ubuntu package.

Running again the script is not really user-friendly. It would be great to have a command like `atuin update`

**Comment by ellie** [comment]
> Script is the recommended way in the documentation

Sure, and if you read the script it will do its very best to install from the system package manager. 

> there is no debian/ubuntu package.

Officially no, as nobody maintains one yet. Most of our users use something other than Ubuntu however

> Running again the script is not really user-friendly. It would be great to have a command like atuin update

Again, while most of our supported systems use a package manager I'd like to avoid updating in place ourselves.

```
bash <(curl https://setup.atuin.sh) && sed -i '$ d' ~/.zshrc
```

There you are. I'm happy to review a PR for `atuin update`, or even for making the script check if the plugin install is needed, but right now it's not a priority, nobody has contributed it, and I'm tired.

Otherwise you can find our ubuntu package here: https://github.com/atuinsh/atuin/releases/download/v16.0.0/atuin_16.0.0_amd64.deb

**Comment by ellie** [comment]
There you are! Updated the install script. Feel free to re-run and it will no longer duplicate the shell setup.

**Comment by baptistebisson** [comment]
> There you are! Updated the install script. Feel free to re-run and it will no longer duplicate the shell setup.

Thanks a lot. I will check if I'm able to add the update command in atuin.
